tests/factories/LocalObservation.js

```diff
@@ -0,0 +1,14 @@
+import { define } from "factoria";
+
+// TODO use faker for more of these dynamic values. Also, would we really
+// store timeObservedAt like this in Realm?!
```

Comment:
Nope, we would store time in the same format as received from the API. Not a very realistic example.